Application/EdFi.Ods.Api/Startup/OdsStartupBase.cs

```diff
@@ -209,10 +209,21 @@ void RegisterModulesDynamically()
         }
 
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
-        public void Configure(IApplicationBuilder app, IWebHostEnvironment env, ILoggerFactory loggerFactory)
+        public void Configure(IApplicationBuilder app, IWebHostEnvironment env, ILoggerFactory loggerFactory, ApiSettings apiSettings)
         {
             loggerFactory.AddLog4Net();
 
+            var virtualPathEnabled = apiSettings.VirtualPathEnabled ?? false;
+
+            if (virtualPathEnabled)
+            {
+                var pathBase = !string.IsNullOrEmpty(apiSettings.PathBase)
```

Comment:
Both here and in the other usage in the controller, we should make sure we're normalizing the path base.  Make sure this handles trailing slash or no trailing slash, leading slash or no leading slash, and make the path base work no matter what.